docs/2-hooks/useCheckoutPricing.stories.mdx

```diff
@@ -0,0 +1,214 @@
+import { Subtitle } from '@storybook/addon-docs/blocks';
+import { PropsTable } from '@storybook/components';
+
+<Meta title="Hooks/useCheckoutPricing" />
+
+# useCheckoutPricing
+
+<Subtitle slot={() => 'To create a pricing preview'} />
+
+`import { useCheckoutPricing } from '@recurly/react-recurly';`
+
+A [custom hook][react-hooks] for interacting with [Recurly.js' pricing API][rjs-pricing] for providing users with an estimate of a purchase before they check out.
+
+To initialize, call `useCheckoutPricing` [with an initial checkout pricing input][use-checkout-pricing-input].
+
+```js
+export default function PricingPreview() {
+  const initialPricingInput = {
+    subscriptions: [
+      {
+        plan: 'my-plan',
+      },
+    ],
+  };
+
+  const [{ price, loading }, setCheckoutPricing] = useCheckoutPricing(initialPricingInput);
+
+  if (!loading) {
+    return <div>{price.now.subtotal}</div>
+  }
+}
+```
+
+Calling `useCheckoutPricing` returns an array of two items:
+
+1. A [useCheckoutPricingState][use-checkout-pricing-state] object for reading output.
+2. A [setCheckoutPricing][set-checkout-pricing] function for updating input values.
+
+For complete documentation on what `useCheckoutPricing` returns, see the [useCheckoutPricingReturn reference page][use-checkout-pricing-return].
+
+## Initializing
+
+It's important to know that immediately after calling `useCheckoutPricing`, `useCheckoutPricingState.price` will default to an empty object while waiting for asynchronous actions to resolve.
+
+The below example will throw a type error because of this.
+```js
+export default function PricingPreview() {
+  const initialPricingInput = {
+    subscriptions: [
+      {
+        plan: 'my-plan',
+      },
+    ],
+  };
+
+  const [{ price }, setCheckoutPricing] = useCheckoutPricing(initialPricingInput);
+
+  return <div>{price.now.subtotal}</div>
+}
+```
+
+The [loading state](#loading-state) can be checked before rendering anything that uses `useCheckoutPricingState.price` to circumvent the error.
+
+```js
+export default function PricingPreview() {
+  const initialPricingInput = {
+    subscriptions: [
+      {
+        plan: 'my-plan',
+      },
+    ],
+  };
+
+  const [{ price, loading }, setCheckoutPricing] = useCheckoutPricing(initialPricingInput);
+
+  if (!loading) {
+    return <div>{price.now && price.now.subtotal}</div>
```

Comment:
```suggestion
    return <div>{price.now?.subtotal}</div>
```

Reply:
Ah yeah this is a mistake. It should just be `price.now.subtotal`, the loading boolean protects the type error from happening.